Migrate Cloudcraft documentation from Help Scout to Datadog (#21358)
DataDog/documentation: content/en/cloudcraft/faq/support-other-cloud-providers.md

---
title: Does Cloudcraft support other cloud providers?

> kind: guide

Reviewer: ```suggestion
kind: faq
```

---

Yes. Cloudcraft supports both AWS and Microsoft Azure.

> We hope to add support for Google Cloud and other providers in the future.

Reviewer: ```suggestion
Support for Google Cloud and other providers will be added at a later date.
```
